finalPrep/README.md

```diff
@@ -45,30 +45,36 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
```

Comment:
Great to see you have some dedicated time to be off computer and eat! I see approximately 16 hours of study time outside of class, which I would say is quite a bit lower than what you can anticipate - so be mentally prepared to add in 6-12 hours in your week.